bin/crossing

```diff
@@ -31,6 +31,7 @@ cmd_opts =
       opt :file, 'the name of the file to retrieve', required: true, type: String
       opt :bucket, 'the bucket the file is stored in', required: true, type: String
       opt :kmskeyid, 'the kms key id of the master KMS key', required: false, type: String
+      opt :binary, 'flag to indicate binary write mode', required: false, default: true
     end
```

Comment:
Will this automatically get passed as true to crossing.get in the case statement if the --binary flag is present? haven't worked much with Trollop